sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachinePopularImageTests.java

```diff
@@ -31,7 +31,9 @@ public void canCreateAllPopularImageVM() {
         rgName = generateRandomResourceName("rg", 10);
         List<Mono<VirtualMachine>> vmMonos = new ArrayList<>();
         for (KnownWindowsVirtualMachineImage image : Arrays.stream(KnownWindowsVirtualMachineImage.values())
-            .filter(image -> image != KnownWindowsVirtualMachineImage.WINDOWS_SERVER_2019_DATACENTER_WITH_CONTAINERS_GEN2 && image != KnownWindowsVirtualMachineImage.WINDOWS_SERVER_2019_DATACENTER_WITH_CONTAINERS)
+            .filter(image -> image != KnownWindowsVirtualMachineImage.WINDOWS_SERVER_2019_DATACENTER_WITH_CONTAINERS_GEN2
+                && image != KnownWindowsVirtualMachineImage.WINDOWS_SERVER_2019_DATACENTER_WITH_CONTAINERS
+                && image != KnownWindowsVirtualMachineImage.WINDOWS_DESKTOP_10_20H1_PRO)
```

Comment:
Does previous api-version fail this test too?

Reply:
If these images no longer supported by Compute, you need to give proper javadoc in code. e.g. https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/KnownLinuxVirtualMachineImage.java#L22-L27
and see if there is a replacement.

Reply:
Yes, the test case `VirtualMachinePopularImageTests#canCreateAllPopularImageVM` fails in `RECORD` and `LIVE` modes.

Reply:
The comment `@Deprecated` has been fixed in the new version.